itest/assets_test.go

```diff
@@ -66,6 +67,242 @@ var (
 	failureNone             = lnrpc.PaymentFailureReason_FAILURE_REASON_NONE
 )
 
+// createTestMultiRFQAssetNetwork creates a lightning network topology which
+// consists of both bitcoin and asset channels. It focuses on the property of
+// having multiple channels available on both the sender and receiver side.
+//
+// The topology we are going for looks like the following:
+//
+//	  /---[sats]--> Erin --[assets]--\
+//	 /                                \
+//	/                                  \
+//
+// Charlie  -----[sats]-->  Dave  --[assets]---->Fabia
+//
+//	\                                  /
+//	 \                                /
+//	  \---[sats]--> Yara --[assets]--/
+func createTestMultiRFQAssetNetwork(t *harnessTest, net *NetworkHarness,
+	charlie, dave, erin, fabia, yara *HarnessNode, charlieTap, daveTap,
+	erinTap, fabiaTap, yaraTap, universeTap *tapClient,
+	mintedAsset *taprpc.Asset, assetSendAmount, fundingAmount uint64,
+	pushSat int64) (*lnrpc.ChannelPoint, *lnrpc.ChannelPoint,
+	*lnrpc.ChannelPoint) {
```

Comment:
this is a lot of arguments. how about using this sort of helper structs to clean this up:
```golang

	type nodeComponents struct {
		Lnd *HarnessNode
		Tapd: *tapClient
	}

	type testFramework struct {
		charlie nodeComponents
		dave nodeComponents
		erin nodeComponents
		fabia nodeComponents
		yara nodeComponents

		universeTap *tapClient
	}
```

and then the rest of the args